joomla/joomla-cms
# [5.1] Convert mod_stats_admin to service provider

### Summary of Changes

Converts the statistics administrator module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_stats_admin published
2. You should see module output

![mod_stats_admin](https://github.com/joomla/joomla-cms/assets/59166858/5f36c669-9014-4b12-9eda-0828eb866a07)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
